src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/EnumConverter.cs

```diff
@@ -24,8 +24,19 @@ internal sealed class EnumConverter<T> : JsonConverter<T>
 
         private readonly JsonNamingPolicy? _namingPolicy;
 
-        private readonly ConcurrentDictionary<ulong, JsonEncodedText> _nameCache;
-
+        /// <summary>
+        /// Holds a mapping from enum value to text that might be formatted with <see cref="_namingPolicy" />.
+        /// </summary>
+        private readonly ConcurrentDictionary<ulong, JsonEncodedText> _namingPolicyCache;
```

Comment:
Nit: this name confuses me since the cache isn't specific to using naming policy
```suggestion
        private readonly ConcurrentDictionary<ulong, JsonEncodedText> _jsonNameCache;
```